### PR TITLE
exit to docs

### PR DESCRIPTION
**Pull request by anki-code**
#3573 #3241

### CODE REVIEW

File: docs/bash_to_xsh.rst [hunk 1]
```diff
@@ -68,6 +68,9 @@ line is ``#!/usr/bin/env xonsh``.
     * - ``$@``
       - ``$ARGS``
       - List of all command line argument and parameter strings.
+    * - ``exit``
+      - ``raise SystemExit`` or ``sys.exit()``
```
Comment (astronouth7303) [inline docs/bash_to_xsh.rst:72]: While `raise SystemExit` does do it, I've literally never seen this in python code. I would not encourage its use.

File: news/exit.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* Added two ways of exit to the "Bash to xsh" article.
```
Comment (astronouth7303) [inline news/exit.rst:7]: FYI, we generally don't require news items for docs, but thank you!

File: docs/bash_to_xsh.rst (hunk 1 above)
Comment (anki-code) [reply docs/bash_to_xsh.rst:72]: 304k in GitHub search - https://github.com/search?q=%22raise+SystemExit%22&type=Code

Comment (astronouth7303) [reply docs/bash_to_xsh.rst:72]: That doesn't mean it's a good idea or idiomatic or something we should actively encourage.

Comment (anki-code) [reply docs/bash_to_xsh.rst]: The best way is to have just `exit()` function by default without any imports. Ok, I remove SystemExit.

File: news/exit.rst (hunk 2 above)
Comment (anki-code) [reply news/exit.rst]: Thank you for info!

**Pull request merged**